scio-core/src/main/scala/com/spotify/scio/io/Tap.scala

```diff
@@ -77,11 +79,12 @@ case class BigQueryTap(table: TableReference) extends Tap[TableRow] {
 }
 
 /** Tap for object files on local file system or GCS. */
-case class ObjectFileTap[T: ClassTag](path: String) extends Tap[T] {
+case class ObjectFileTap[T: ClassTag](path: String, coder: Coder[T] = null) extends Tap[T] {
```

Comment:
We're not overriding `coder` yet and it's not exposed in `ScioContext` or `SCollection`? Remove the optional arg for now?